### PR TITLE
ORC-607: Sync orc-benchmarks module to the others

### PR DESCRIPTION
**Pull request by pgaref**

### CODE REVIEW

**Comment by dongjoon-hyun** [comment]
Thank you for making a PR, @pgaref .

File: java/bench/pom.xml [hunk 1]
```diff
@@ -41,11 +41,11 @@
     <hive.version>2.3.3</hive.version>
```
Comment (dongjoon-hyun) [inline java/bench/pom.xml]: Shall we bump this to the latest `2.3.6`?

File: java/bench/pom.xml [hunk 2]
```diff
@@ -41,11 +41,11 @@
...
     <iceberg.version>0.1.3</iceberg.version>
     <jmh.version>1.20</jmh.version>
-    <orc.version>1.5.2</orc.version>
+    <orc.version>${project.version}</orc.version>
     <parquet.version>1.8.3</parquet.version>
     <slf4j.version>1.7.25</slf4j.version>
     <spark.version>2.4.0</spark.version>
```
Comment (dongjoon-hyun) [inline java/bench/pom.xml]: Can we have `2.4.5`?

File: java/bench/pom.xml [hunk 3]
```diff
@@ -41,11 +41,11 @@
     <hive.version>2.3.3</hive.version>
     <iceberg.version>0.1.3</iceberg.version>
     <jmh.version>1.20</jmh.version>
```
Comment (dongjoon-hyun) [inline java/bench/pom.xml]: `1.23`?

**Comment by dongjoon-hyun** [comment]
While revisting this again, I noticed that the original scope is not `Bump orc-benchmarks dependencies`. This seems to aiming to have the same SNAPSHOT version number. I'll update the PR title.

**Comment by pgaref** [comment]
> While revisting this again, I noticed that the original scope is not `Bump orc-benchmarks dependencies`. This seems to aiming to have the same SNAPSHOT version number. I'll update the PR title.

Thanks for clearing this out! We can update project dependencies such as Spark and Hive in a following PR.

**Pull request closed without merging**